WordPress/gutenberg
# Remove core/post-comments-form block styles that are overriding button element styles

## What?

Remove core/post-comments-form block styles that are overriding button element styles from `theme.json`

## Why?


These styles are preventing that the `theme.json` styles for button elements apply over the core/post-comments-form submit button.

## How?

I think those styles can be removed because they are no longer needed and they are conflicting with the button element styles.

## Testing Instructions





1. Add theme.json settings for core/button block:
```
	"styles": {
              "elements": {
                      "button": {
                              "border": {
	                              "color": "black",
	                              "style": "solid",
	                              "width": "3px"
                              }
              },
	...
```
2. Check the rendered post comments form and see if the styles are being overridden.

## Screenshots or screencast 

Using the `theme.json` from the top :arrow_up:  example:

**Before:**
![image](https://user-images.githubusercontent.com/1310626/176474203-7da7490c-a258-4b5e-9591-3b4f6ce9ee1b.png)


**After:**
![image](https://user-images.githubusercontent.com/1310626/176473877-9d8b2ea4-68b8-4b87-8fa9-aeea715da417.png)
